overlapping_panels/src/main/java/com/discord/panels/OverlappingPanelsLayout.kt

```diff
@@ -753,13 +753,13 @@ open class OverlappingPanelsLayout : FrameLayout {
     }
 
     startPanelState = getStartPanelState(previousX, x)
-    for (leftPanelStateListener in startPanelStateListeners) {
-      leftPanelStateListener.onPanelStateChange(startPanelState)
+    for (startPanelStateListener in startPanelStateListeners) {
```

Comment:
The changes in this file are just variable renames from "left" / "right" to "start" / "end", since `OverlappingPanelsLayout` now support right-to-left layout directions.